content/sensu-go/5.16/api/filters.md

```diff
@@ -105,12 +138,12 @@ payload         | {{< highlight shell >}}
   },
   "action": "deny",
   "expressions": [
-    "event.entity.metadata.namespace == 'production'"
+    "event.entity.metadata.namespace == '\''production'\''"
```

Comment:
This doesn't seem quite right to my eyes...are these meant to be escaped?

Reply:
I used Postman's copy button to copy the code, so it might be something Postman adds? In any case, I will remove it.